### ISSUE TITLE
Training example

### ISSUE THREAD

**Issue by robclouth** [lucidrains/vector-quantize-pytorch#1]
Hey, do you have an example of how to train this?

**Comment by robclouth** [comment]
You just repeated push x in until the loss stops decreasing.
